types/woocommerce__woocommerce-rest-api/index.d.ts

```diff
@@ -0,0 +1,124 @@
+// Type definitions for @woocommerce/woocommerce-rest-api 1.0
+// Project: https://github.com/woocommerce/woocommerce-rest-api-js-lib
+// Definitions by: KaKa <https://github.com/climba03003>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+export type WooCommerceRestApiVersion = 'wc/v3' | 'wc/v2' | 'wc/v1' | 'wc-api/v3' | 'wc-api/v2' | 'wc-api/v1';
+export type WooCommerceRestApiEncoding = 'utf-8' | 'ascii';
+export type WooCommerceRestApiMethod = 'get' | 'post' | 'put' | 'delete' | 'options';
+
+export interface WooCommerceRestApiOptions {
+    /* Your Store URL, example: http://woo.dev/ */
+    url: string;
+    /* Your API consumer key */
+    consumerKey: string;
+    /* 	Your API consumer secret */
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/woocommerce__woocommerce-rest-api/index.d.ts;
  Line: 14;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.